finalPrep/modZeroHero.js

```diff
@@ -2,48 +2,100 @@
 
 // Declare two variables - heroName AND specialAbility - set to strings
 
+var heroName = "Superman";
+var specialAbility = "flying";
+
 // Declare two variables - greeting AND catchphrase
 //   greeting should be assigned to a string that uses concatenation to include the heroName
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
 
+var greeting = "Hello it's me, " + heroName;
+var catchphrase = "Have no fear, " + heroName + " is here!";
+
 // Declare two variables - power AND energy - set to integers
 
+var power = 1000;
+var energy = 1000;
+
 // Declare two variables - fullPower AND fullEnergy
 //   fullPower should multiply your current power by 500
 //   fullEnergy should add 150 to your current energy
 
+var fullPower = power * 500;
+var fullEnergy = power + 150;
+
 // Declare two variables - isHuman and identityConcealed - assigned to booleans
 
+var isHuman = false;
+var identityConcealed = true;
 
 // Declare two variables - archEnemies AND sidekicks
 //   archEnemies should be an array of at least 3 different enemy strings
 //   sidekicks should be an array of at least 3 different sidekick strings
 
+var archEnemies = ["Lex Luther", "General Zod", "Brainiac"];
+var sidekicks = ["Batman", "Justice League", "Krypto the Superdog"];
+
 // Print the first sidekick to your console
+console.log(sidekicks[0]);
 
 // Print the last archEnemy to the console
+console.log(archEnemy[2]);
 
 // Write some code to add a new archEnemy to the archEnemies array
+let newLength = archEnemies.push("Darkseid");
 
 // Print the archEnemies array to console to ensure you added a new archEnemy
 
+console.log(archEnemies);
+
 // Remove the first sidekick from the sidekicks array
 
+let first = sidekicks.shift([0]);
+
 // Print the sidekicks array to console to ensure you added a new sidekick
+console.log(sidekicks);
 
 // Create a function called assessSituation that takes three arguments - dangerLevel, saveTheDay, badExcuse
 //   - dangerLevel should be an integer
-//   - saveTheDay should be a string a hero would say once they save the day 
+//   - saveTheDay should be a string a hero would say once they save the day
 //   - badExcuse should be a string a hero would say if they are too afraid of the dangerLevel
 
+function assessSituation (dangerLevel, saveTheDay, badExcuse) {
+  var saveTheDay = "Crime won't stand a chance against me!";
+  var badExcuse = "I'm scared, I've never gone up against this many villians";
+  var dangerLevel = dangerLevel;
+  if ((10 < dangerLevel) && (dangerLevel < 50)) {
```

Comment:
Nice job here. As you build conditionals, think about trying to put the most complex logic in your `else` block when possible.